### PR TITLE
chore(hound_ci): integrate hound CI

### PR DESCRIPTION
**Pull request by OKiMaureen**
#### What does this PR do?
Integrate hound ci
#### Description of Task to be completed?
- add hound.yml file
- add hound ci badge to readme
#### How should this be manually tested?
- Check the badge in readme
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/159232030

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
Comment (Oloyedesinmiloluwa) [inline .hound.yml]: Good job, but you need to add a new line at the end of this file

Comment (Oloyedesinmiloluwa) [inline .hound.yml]: Good job, but you need to add a new line at the end of this file

Comment (anneKay) [reply .hound.yml]: Hound CI is working fine, good job

Comment (OKiMaureen) [reply .hound.yml]: @Oloyedesinmiloluwa, Thank you for the feedback, I have addressed the issue you raised.

**Pull request merged**